migration/src/m0000585_analysis_api_index.rs

```diff
@@ -0,0 +1,46 @@
+use sea_orm_migration::prelude::*;
+
+#[derive(DeriveMigrationName)]
+pub struct Migration;
+
+#[async_trait::async_trait]
+#[allow(deprecated)]
+impl MigrationTrait for Migration {
+    async fn up(&self, manager: &SchemaManager) -> Result<(), DbErr> {
+        manager
+            .get_connection()
+            .execute_unprepared(
+                r#"
+                CREATE EXTENSION IF NOT EXISTS pg_trgm;
```

Comment:
question: does this work with the embedded pgsql in PM mode?

Reply:
yes